components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/display/DisplayToolbar.kt

```diff
@@ -460,6 +463,8 @@ class DisplayToolbar internal constructor(
         }
 
         views.securityIndicator.siteSecurity = siteSecurity
+        views.securityIndicator.contentDescription =
+            context.getString(R.string.mozac_browser_toolbar_content_description_site_info)
```

Comment:
Why are we applying the content description twice? Can we just set it in the layout XML?

Reply:
Sorry, I've made some changes, cause just with the second setting of content description the test was not passing.
I've changed the commit to apply description in xml.